spec/requests/sessions_spec.rb

```diff
@@ -7,7 +7,9 @@
       get(auth_dfe_callback_path)
 
       get(signout_path)
-      expect(response).to redirect_to("#{Settings.dfe_signin.issuer}/session/end?id_token_hint=&post_logout_redirect_uri=https://localhost:3000/auth/dfe/signout")
+      expect(response).to redirect_to(
+                            "#{Settings.dfe_signin.issuer}/session/end?id_token_hint=&post_logout_redirect_uri=https%3A%2F%2Flocalhost%3A3000%2Fauth%2Fdfe%2Fsignout"
```

Comment:
"#{Settings.dfe_signin.issuer}/session/end?id_token_hint=`MISSING???`&post_logout_redirect_uri=https%3A%2F%2Flocalhost%3A3000%2Fauth%2Fdfe%2Fsignout"

missing `id_token_hint` and url encoding?

Reply:
`id_token_hint` is in both the old and the new there, the diff is the URL encoding, which is just a side-effect of cleaning up `SessionsController#signout` because I couldn't read what it was doing. We probably really do want to URL encode the URL being included as a parameter in another URL, turns out, so net win.

Reply:
misha net win go go go :+1: